nixos/maintainers/option-usages.nix

```diff
@@ -143,36 +142,33 @@ let
     }) introspectionModules;
 
   displayOptionsGraph =
-     let
-       checkList =
-         if testOption != null then [ testOption ]
-         else testOptions;
-       checkAll = checkList == [];
-     in
-       flip filter graph ({option, ...}:
-         (checkAll || elem option checkList)
-         && !(elem option excludedTestOptions)
-       );
+    let
+      checkList =
+        if testOption != null then [ testOption ]
+        else testOptions;
+      checkAll = checkList == [ ];
+    in
+    flip filter graph ({ option, ... }:
+      (checkAll || elem option checkList) && !(elem option excludedTestOptions)
+    );
 
   graphToDot = graph: ''
     digraph "Option Usages" {
-      ${concatMapStrings ({option, usedBy}:
-          concatMapStrings (user: ''
-            "${option}" -> "${user}"''
-          ) usedBy
-        ) displayOptionsGraph}
+      ${concatMapStrings ({ option, usedBy }:
+      concatMapStrings (user: ''
+        "${option}" -> "${user}"''
+        ) usedBy
+    ) displayOptionsGraph}
     }
   '';
 
   graphToText = graph:
-    concatMapStrings ({usedBy, ...}:
-        concatMapStrings (user: ''
-          ${user}
-        '') usedBy
-      ) displayOptionsGraph;
-
+    concatMapStrings ({ usedBy, ... }:
+      concatMapStrings (user: ''
+        ${user}
+      '') usedBy
+    ) displayOptionsGraph;
```

Comment:
I need to reconsider on how nixpkgs-fmt approach this syntax.

Reply:
the `graphToText` is fine, but the previous change that caused some weird result (`graphToDot')